WordPress/Tests/NamingConventions/PrefixAllGlobalsUnitTest.3.inc

```diff
@@ -21,4 +21,35 @@ $acronym_test = new class extends \PHPUnit_Framework_TestCase {
 	}
 };
 
+class Some_Test extends WP_Font_Face_UnitTestCase {
```

Comment:
I think it would be helpful to have a comment above this test case to explain what's being tested here too. Or maybe change the class name to be descriptive ?

The way it reads now, this test is grouped with the first two (no false positives) tests, while it is a _no false negative_ test.